pkg/base/config.go

```diff
@@ -633,10 +633,10 @@ func (cfg *RaftConfig) SetDefaults() {
 		cfg.RaftTickInterval = defaultRaftTickInterval
 	}
 	if cfg.RaftElectionTimeoutTicks == 0 {
-		cfg.RaftElectionTimeoutTicks = defaultRaftElectionTimeoutTicks
+		cfg.RaftElectionTimeoutTicks = uint64(defaultRaftElectionTimeoutTicks)
```

Comment:
Can we change the type of `defaultRaftElectionTimeoutTicks` and `defaultRaftHeartbeatIntervalTicks`? `EnvOrDefaultInt64` exists, if you agree with my other comment.